### PR TITLE
Markdown Clean Up

### PR DESCRIPTION
**Pull request by fallengravity**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,127 +3,139 @@
...
+
+-   [Application](https://github.com/lithe-project/awesome-cryptonote#application)
+-   [Blockchain](https://github.com/lithe-project/awesome-cryptonote#blockchain)
+-   [Documentation](https://github.com/lithe-project/awesome-cryptonote#documentation)
+-   [Graphical User Interface (GUI)](https://github.com/lithe-project/awesome-cryptonote#graphical-user-interfaces-gui)
+-   [Hashing Algorithm (PoW)](https://github.com/lithe-project/awesome-cryptonote#hashing-algorithm)
+-   [Mining Software](https://github.com/lithe-project/awesome-cryptonote#mining-software)
+-   [Pool Software](https://github.com/lithe-project/awesome-cryptonote#pool-software)
+-   [Web Wallet](https://github.com/lithe-project/awesome-cryptonote#web-wallet)
+-   [Misc.](https://github.com/lithe-project/awesome-cryptonote#misc)
```
Comment (LithyRiolu) [inline README.md]: Can you revert this bit, please?

Currently, the layout of the document is set out like;
- title
    - subtitle
    - subtitle

but the new solution makes them all titles

File: README.md [hunk 2]
```diff
@@ -3,127 +3,139 @@
...
 ### Misc.
-*Applications, games, resources and other things built and/or maintained by CryptoNote developers.*
 
-- [DecentraChat](https://github.com/lithyriolu/decentrachat) - [LithyRiolu](https://github.com/lithyriolu) - [The Lithe Project&trade;](https://github.com/lithe-project)
+_Applications, games, resources and other things built and/or maintained by CryptoNote developers._
 
-***
+-   [DecentraChat](https://github.com/lithyriolu/decentrachat)
+-   [LithyRiolu](https://github.com/lithyriolu)
+-   [The Lithe Project™](https://github.com/lithe-project)
```
Comment (LithyRiolu) [inline README.md]: And these 3 new lines too.

The layout will look like this in the future hopefully
- Appname - Author - Project
- Appname - Author - Project

**Review by LithyRiolu — CHANGES_REQUESTED** [review]
It looks great, thanks for cleaning up the markdown to its correct syntax. Just 2 things need changing before merge.

**Comment by fallengravity** [comment]
There are more commits there than I wanted, git was fucking me around, anyway, issues are resolved.

**Comment by LithyRiolu** [comment]
Thanks for your time to fix it, great contribution!

**Pull request merged**